webapp/Makefile

```
@@ -0,0 +1,13 @@
+.PHONY: help
+.DEFAULT_GOAL := help
+
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+setup-env:					## Setup the environment
+	cp .env.dist .env
+
+install: setup-env			## Install dependencies
+	composer install
```

Some more nitpicking: putting `setup-env` as  dependency of `install` is a good idea, however you may want to change `setup-env` to the following, which will avoid overriding the current config if there is already one:

```suggestion
setup-env:					## Setup the environment
	cp -n .env.dist .env

install: setup-env			## Install dependencies
	composer install
```